tests/unit/drivers/test_kv_index_driver.py

```diff
@@ -18,6 +18,14 @@ def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
         for key, value in zip(keys, values):
             self.docs[key] = value
 
+    def update(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
```

Comment:
u can pass with these, I think these 2 methods won't be used

Reply:
They are used here: `SimpleKVIndexDriver(method='update')`
However, I van remove the other methods: query, get_query_handler...